### ISSUE TITLE
Configurable Tiddler Width via Theme Tweaks in Control Panel

### ISSUE THREAD

**Issue by keelancook** [crazko/krystal#7]
It would be great if we could adjust the width of all panes via a setting in the control panel. There is an option for theme tweaks, and you've already created a tab for Krystal, so it would make sense to allow the pixel width for the panes to be set there without having to edit a shadow tiddler.

**Comment by crazko** [comment]
Had some time, so it's implemented now. Please, give it a try and let me know.

**Comment by keelancook** [comment]
Definitely a step in the right direction, per the configurable menu items. However, it's not jiving for some reason with my setup. The control panel tiddler is now very very wide, as in wider than my monitor, and it does not appear to be affected by changing the tiddler width. In addition, it seems the change to width does not always work. Some tiddlers remain varied lengths.

**Comment by crazko** [comment]
What other plugins do you use? Also, can you check you don't have any shadow tiddlers created?

**Comment by keelancook** [comment]
It doesn't appear to be a direct conflict with any of the additional plugins. I took a clean empty of yours and installed all of the plugins into that one and it works fine. 

FYI, though, I'm currently using relink, edit-comptext, Twitter, refnotes, TWCrosslinks, and Streams plugins. You might want to look into those last three, as I think they greatly enhance the value of your theme, especially Streams. 

I can't find a shadow tiddler or anything that should be causing the conflict, but I'm sure it is something like that. For my part, I just imported all of my content Tiddlers into a fresh copy of yours with my other plugins and I'm all set. It's working as intended.

I do wish I could help you isolate what caused it to break with my original setup, but it had to be some rogue tiddler I had created on accident. After using a clean install, the ability to easily configure tiddler width and padding from the control panel (as well as the link highlighting) are great. 

Thanks again!
